.c8rc-esm-unit.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Would it be better to put these in a folder so they're not polluting the top-level structure?

Reply:
I could. They are dot files and typically hidden and we don't publish these with the agent

Reply:
my other option could be to just keep 1 config for the common stuff and pass in the reports dir in package.json when calling c8

Reply:
They do show up on github though, which is basically this agent's "homepage". I don't feel so strongly enough to really push it further than this though, so I'll leave it to you :)

Reply:
> my other option could be to just keep 1 config for the common stuff and pass in the reports dir in package.json when calling c8

I like this idea!